gef.py

```diff
@@ -10889,6 +10890,62 @@ def screen_setup(self) -> None:
         return
 
 
+class GefInstallExtraScriptCommand(gdb.Command):
+    """`gef install` command: installs one or more scripts from the `gef-extras` script repo. Note that the command
+    doesn't check for external dependencies the script(s) might require."""
+    _cmdline_ = "gef install"
+    _syntax_  = f"{_cmdline_} SCRIPTNAME [SCRIPTNAME [SCRIPTNAME...]]"
+
+    def __init__(self) -> None:
+        super().__init__(self._cmdline_, gdb.COMMAND_SUPPORT, gdb.COMPLETE_NONE, False)
+        self.branch = gef.config.get("gef.default_branch", "master")
```

Comment:
```suggestion
        self.branch = gef.config.get("gef.extras_default_branch", "master")
```

I think it is clearer this way that this config refers to gef-extras instead of gef itself. If you take this suggestion, remember to update the docs too.